### ISSUE TITLE
Accessibility Issue - Language is missing/invalid in Project view (Umbraco Cloud)

### ISSUE THREAD

**Issue by Truealex4** [umbraco/Umbraco-CMS#11910]
### Which *exact* Umbraco version are you using? For example: 9.0.1 - don't just write v9

9.0.1 of Cloud

### Bug summary

When I view the Project and subsequent Project child page (ie where you can specify which environment you wish to use) The <html lang> attribute is missing or is empty, or a lang attribute value is not a valid language identifier.

This issue was revealed by running WAVE (Web accessibility evaluation tool) provided by 'Webaim'.

**Example area of code affected**:
html ng-app="uaas" class="ng-scope" style="margin-left: 380px;">
<head>
<style>
@charset "UTF-8";[ng\:cloak],[ng-cloak],[data-ng-cloak],[x-ng-cloak],.ng-cloak,.x-ng-cloak,.ng-hide:not(.ng-hide-animate){display:none !important;}ng\:form{display:block;}.ng-animate-shim{visibility:hidden;}.ng-anchor{position:absolute;}
</style>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
<title>

![image](https://user-images.githubusercontent.com/98405431/151159126-a2df3b22-ff12-4d66-842b-1b056536064b.png)

### Specifics

https://www.s1.umbraco.io/projects
And subsequent project child pages ie https://www.s1.umbraco.io/project/my-website (fictional example)

### Steps to reproduce

1. Install WAVE accessibility tool as chrome extension.
2. Access Umbraco Login and enter valid credentials in chrome.
3. Click on the error that is returned in the WAVE 'summary'.

### Expected result / actual result

**EXPECTED RESULT**
No WAVE errors returned in Umbraco.

**ACTUAL RESULT**
The following error is present:
The <html lang> attribute is missing or is empty, or a lang attribute value is not a valid language identifier.

**Comment by Bakersbakebread** [comment]
Can also reproduce this on cloud dashboard, no lang attribute on html element
